CMakeLists.txt

```diff
@@ -101,6 +101,11 @@ target_link_libraries(HighFive INTERFACE HighFive::Include)
 if(HIGHFIVE_FIND_HDF5)
   find_package(HDF5 REQUIRED)
   target_link_libraries(HighFive INTERFACE HDF5::HDF5)
+  
+  # I install hdf5-blosc to the default path
+  set(blosc_filter_DIR /usr/local/lib/)
+  target_link_libraries(HighFive INTERFACE blosc_filter)
+  target_link_directories(HighFive INTERFACE /usr/local/include/)
```

Comment:
We will have to fix that.

Reply:
Since blosc is an extra plugin of hdf5, it is necessary to compile and link this external libraries. hdf5-blosc can not be found using find_package macro since it is packaged too simple [here](https://github.com/Blosc/hdf5-blosc/blob/master/CMakeLists.txt#L75). I haven't figured out an elegant way to link this library.

Reply:
What I mean is that hard coded path to your system is not the valid way to give access to blosc.